controllers/openstackvmset_controller.go

```diff
@@ -223,7 +226,7 @@ func (r *OpenStackVMSetReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	}
 
 	// If we determine that a backup is overriding this reconcile, requeue after a longer delay
-	overrideReconcile, err := ospdirectorv1beta1.OpenStackBackupOverridesReconcile(r.Client, instance.Namespace, instance.Status.ProvisioningStatus.State == ospdirectorv1beta1.VMSetCondTypeProvisioned)
+	overrideReconcile, err := ospdirectorv1beta1.OpenStackBackupOverridesReconcile(r.Client, instance.Namespace, instance.Status.ProvisioningStatus.State == ospdirectorv1beta1.VMSetCondTypeProvisioned || instance.Status.ProvisioningStatus.State == ospdirectorv1beta1.VMSetCondTypeEmpty)
```

Comment:
as we discussed a while back to get rid of the dedicated ProvisioningStatus, could use cond here which is a this point currentCondition?

Reply:
That is the proper approach.  Will fix.

Reply:
Done